src/tools/attack/attack.cpp

```diff
@@ -22,7 +22,9 @@
 
 #define ATTACK_TIMEOUT_DEFAULT_MS (60 * 1000)
 
-#define ATTACK_THREADS_DEFAULT CxPlatProcCount()
+#define ATTACK_THREADS_DEFAULT max(1, CxPlatProcCount())
```

Comment:
This should be unnecessary. Processor count will never be zero

Reply:
Good point.